doc/Pacemaker_Remote/en-US/Ch-KVM-Tutorial.txt

```diff
@@ -459,9 +461,29 @@ Failed actions:
     guest1_monitor_30000 (node=example-host, call=3, rc=7, status=complete): not running
 ----
 
-=== Accessing Cluster Tools from Remote-node ===
-
-Besides just allowing the cluster to manage resources on a remote-node, pacemaker_remote has one other trick.  +The pacemaker_remote daemon allows nearly all the pacemaker tools (crm_resource, crm_mon, crm_attribute, crm_master) to work on remote nodes natively.+
-
-Try it, run +crm_mon+ or +pcs status+ on the guest after pacemaker has integrated the remote-node into the cluster.  These tools just work.  These means resource agents such as master/slave resources which need access to tools like crm_master work seamlessly on the remote-nodes.
-
+=== Accessing Cluster Tools from Guest Node ===
+
+Besides allowing the cluster to manage resources on a guest node,
+pacemaker_remote has one other trick. The pacemaker_remote daemon allows
+nearly all the pacemaker tools (`crm_resource`, `crm_mon`, `crm_attribute`,
+`crm_master`, etc.) to work on guest nodes natively.
+
+Try it: Run `crm_mon` or `pcs status` on the guest after pacemaker has
+integrated the guest node into the cluster. These tools just work. This
+means resource agents such as master/slave resources which need access to tools
+like `crm_master` work seamlessly on the guest nodes.
+
+[NOTE]
+======
+It is possible to run `pacemaker_remote` inside an LXC container instead of
+a virtual machine, following a similar process. This approach is deprecated
```

Comment:
I'd also move it back to the Ch-LXC-Tutorial.txt file rather than hiding it in a Note block.

You can leave a little bit more of the descriptive parts of that file too.